addons/api/CfgModules.hpp

```diff
@@ -9,9 +9,9 @@ class CfgVehicles
 {
     class Logic;
     class Module_F: Logic { class ArgumentsBaseUnits { class Units; }; class ModuleDescription { class AnyBrain; }; };
-    
+
     class GVAR(basicMissionSetup) : Module_F {
-        scope = 2;
+        scope = 1;
```

Comment:
I would disagree with hiding the modules for now. As they are pretty intuitive for most mission makers